chirp/drivers/uv5r.py

```diff
@@ -602,7 +602,7 @@ def _do_download(radio):
             data += _read_block(radio, i, 0x40, False)
 
     if append_model:
-        data += radio.MODEL.ljust(8)
+        data += radio.MODEL.ljust(8).encode('ascii', 'ignore')
```

Comment:
Can you make this just `.encode()`? We shouldn't have any non-ASCII characters in those strings, since they're used for display to the user. This `ignore` flag will prevent us from catching errors if that ever changes.

Reply:
Changed it and made sure that it still works.

Reply:
Thanks, I'll merge this in a minute assuming tests pass.